settings.gradle

```diff
@@ -17,30 +17,74 @@
 
 rootProject.name = "solr-root"
 
-includeBuild("dev-tools/missing-doclet")
+includeBuild("dev-tools/solr-missing-doclet")
 
-if (!(System.getProperty("skip.solr") as Boolean)) {
```

Comment:
This is vestigial, from the previous joint build, now obviated by the lucene/solr project split?

Reply:
Yes, this should actually be applied to main too (maybe cherry pick the commit).